domain/src/main/kotlin/no/nav/su/se/bakover/domain/vedtak/Vedtak.kt

```diff
@@ -137,32 +124,13 @@ sealed class Vedtak : VedtakFelles, Visitable<VedtakVisitor> {
         override val behandling: Behandling,
         override val saksbehandler: NavIdentBruker.Saksbehandler,
         override val attestant: NavIdentBruker.Attestant,
-        override val journalføringOgBrevdistribusjon: JournalføringOgBrevdistribusjon,
```

Comment:
På generell basis (dvs. utenfor denne PR): Jeg savner en referanse til dokument fra de vedtakene som skal eller kan ha utgående dokumenter. Jeg gjetter på at grunnen til at den ikke er med er pre-optimalisering? Typen hadde ikke trengt å inneholde selve blobben, men kun metadata. Jeg skjønner det sånn at vi ikke bruker den metadataen per tidspunkt til noe, men hvis vi skal begynne med det, mener jeg den referansen hører hjemme her.

Reply:
Bakgrunnen for å ikke ha referansen her var vel i utganspunktet ikke tenkt på som en optimalisering, heller en forenkling som gjør at "alle" domeneobjektene slipper å legge til et nytt objekt for noe som kan/ikke kan eksistere (vedtak er sannsynligvis greie nok, revurderinger er kanskje litt værre mtp forhåndsvarsling etc). Metadata benyttes i dag for å hente ut alle dokumenter knyttet til et vedtak, og det ligger til grunn en antakelse om at det kommer et ønske om å kunne se alle dokumenter for en sak, revurderinge etc etterhvert.